nodejs/week1/src/backend/index.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express");
+const meals = require("./data/meals.json");
+const reservations = require("./data/reservations.json");
+
+const app = express();
+
+app.get("/meals", (req, res)=>{
+    res.json(meals);
+});
+
+app.get("/reservations", (req, res)=>{
+    res.json(reservations);
+});
+
+app.get("/cheap-meals", (req, res)=>{
+    cheapMeals = meals.filter((cheapmeal)=>{
```

Comment:
Great job! 

Some improvements:

1) You declare a variable with the let/const/var keywords. In this case since these variable never changes, the appropriate keyword is const:

`const cheapMeals`

2) You can simplify the arrow function as:

meals.filter(cheapMeal => cheapMeal.price < 65)

This comes really handy when you have a simple arrow function that just returns the value of an expression. The thing on the right of the arrow function is returned implicitly.

Reply:
An additional comment. In:

`meals.filter(cheapMeal => cheapMeal.price < 65)`

the `cheapMeal` is not yet a cheap meal, it is a meal. The `cheapMeal` variable takes the value of every meal in the meals array. The result of evaluating is indeed a cheap meal because it has satisfied the condition of being lower than 65. Therefore, a more descriptive solution would be to write:

```
const cheapMeals = meals.filter(meal => meal.price < 65);
           /                      \
these are cheap          this takes every meal
because they                so not cheap yet, until
have been                         we make sure its price is lower than 65
filtered
```